EasyPost.Tests/AddressTest.cs

```diff
@@ -1,147 +1,127 @@
 using System.Collections.Generic;
 using System.Threading.Tasks;
-using Microsoft.VisualStudio.TestTools.UnitTesting;
+using EasyPost.Models.API;
+using Xunit;
 
 namespace EasyPost.Tests
 {
-    [TestClass]
-    public class AddressTest
+    public class AddressTest : UnitTest
     {
-        private TestUtils.VCR _vcr;
-
-        [TestInitialize]
-        public void Initialize()
-        {
-            _vcr = new TestUtils.VCR("address");
-        }
-
-        private static async Task<Address> CreateBasicAddress()
-        {
-            return await Address.Create(Fixture.BasicAddress);
-        }
-
-        [TestMethod]
-        public async Task TestCreate()
+        public AddressTest() : base("address")
         {
-            _vcr.SetUpTest("create");
-
-            Address address = await CreateBasicAddress();
-
-            Assert.IsInstanceOfType(address, typeof(Address));
-            Assert.IsTrue(address.id.StartsWith("adr_"));
-            Assert.AreEqual("388 Townsend St", address.street1);
         }
 
-        [TestMethod]
-        public async Task TestCreateVerify()
+        [Fact]
+        public async Task TestAll()
```

Comment:
Question: Can we keep these CRUD ordered so they match the other libs? Otherwise there will be a bit more searching to be done to find stuff. 

This is true of the order of all tests.

Reply:
Resolved using annotations and file layout rules. These will automatically be sorted and remain sorted during the code formatting process moving forward.